cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/ResetPasswordRequestController.java

```diff
@@ -0,0 +1,14 @@
+package edu.asu.diging.cord19.explorer.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class ResetPasswordRequestController {
+
+    @RequestMapping(value="/reset/password/request")
+    public String show(Model model) {
+        return "resetPassword";
+    }
+}
```

Comment:
I believe this controller is provided by the simple user plugin?
https://github.com/diging/simple-users/tree/develop/simple-users/src/main/java/edu/asu/diging/simpleusers/web

Reply:
I will remove these controllers and see if it works.

Reply:
you probably have to configure it with the urls